### PR TITLE
Adds an option to disable gay music

### PR DESCRIPTION
**Pull request by ghost**
:cl: Thefastfoodguy
add: Adds an option to disable gay music
/:cl:

### CODE REVIEW

**Comment by PopNotes** [comment]
pretty dumb

**Comment by ghost** [comment]
@PopNotes this is so i dont have to listen to Lean music

**Comment by dreamcrab** [comment]
-5 no thanks

**Comment by ghost** [comment]
@dreamcrab could you say why rather than leaving stupid comments

**Comment by dreamcrab** [comment]
because -5

**Comment by ghost** [comment]
@dreamcrab give an actual reason, fucko

**Comment by ghost** [comment]
why should i have to inconvenience myself just to not hear gay music that shouldn't be in at all

**Comment by dreamcrab** [comment]
because its part of the chemical that i approved in the first place?

**Comment by ghost** [comment]
@dreamcrab that part serves no purpose other than being annoying as fuck and nearly unmuteable

**Comment by dreamcrab** [comment]
humour

**Comment by dreamcrab** [comment]
-5

**Comment by ghost** [comment]
@dreamcrab either way it rapes my ears

**Comment by Pyko1** [comment]
Honestly just make a fix if its earrape

**Comment by dreamcrab** [comment]
its not ear rape he just doesnt like it

**Comment by Pyko1** [comment]
welp, maybe nerf chempiles instead, so that drinks spilled on floor dont affect you?

I mean anyone who had went to club would be already dead if RL would be spessmen13.
Imho all shoes should protect from ingesting chempiles, so you would need acid or something to actually to get the shit in to people.

**Comment by dreamcrab** [comment]
yea, i was actually going to suggest that shoes block it entirely. barefeet could let it happen maybe

File: code/modules/client/preferences_savefile.dm [hunk 1]
```diff
@@ -185,7 +185,9 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	menuoptions		= SANITIZE_LIST(menuoptions)
 	be_special		= SANITIZE_LIST(be_special)
 	pda_style		= sanitize_inlist(MONO, VT, SHARE, ORBITRON)
-
+	
+	hippie_load_preferences(S)
```
Comment (Carbonhell) [inline code/modules/client/preferences_savefile.dm:189]: Didnt i already make this proc for loadouts

**Comment by Spacedong** [comment]
its a gameplay feature, if you don't want it, don't fucking make lean, this just puts players that didn't mute it at an advantage, because they'll still hear you making noise

**Comment by Spacedong** [comment]
grudgecode btw

**Comment by ghost** [comment]
Stale PR with open conflicts, unlikely to get merged anyway, closing for now

**Pull request closed without merging**